target/classes/liquibase/scripts/notification_task.sql

```diff
@@ -0,0 +1,10 @@
+-- liquibase formatted sql
+
+-- changeset yarek2003:1
+create table notification_task(
+                                  notification_task_id serial primary key ,
+                                  chat_id serial not null ,
```

Comment:
```suggestion
                                  chat_id bigint not null ,
```
тип должен быть bigint, так как в сущности используешь Long